Makefile

```diff
@@ -141,6 +127,12 @@ tailscale-test.apk: gradle-dependencies
 	(cd android && ./gradlew assembleApplicationTestAndroidTest)
 	install -C ./android/build/outputs/apk/androidTest/applicationTest/android-applicationTest-androidTest.apk $@
 
+tailscale.version: go.mod go.sum
+    $(shell ./tool/go run tailscale.com/cmd/mkversion > tailscale.version)
+
```

Comment:
oh, so we're still doing the full git clone pain, just hidden behind this tool

Reply:
yeah, at least we now are down to one implementation of that behavior though!

Reply:
hah. lolsob.